libs/blocks/locui-create/create/view.js

```diff
@@ -0,0 +1,5 @@
+import { html } from '../../../deps/htm-preact.js';
+
+export default function Create({}) {
```

Comment:
If `Create` element is not taking any props, then probably we could remove `{}`

Reply:
Added new line char